### PR TITLE
VP-1233 Fix visual discrepancies

### PR DESCRIPTION
**Pull request by cedricpim**
- [x] Correct colors
- [x] Correct text color on hover for outline
- [x] Correct hover color
- [x] Correct size of button when there is an icon
- [x] Rounded corners for consistency

image

### CODE REVIEW

File: src/ui/VavatoTheme.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const VavatoTheme = {
-  primary: '#a29060',
-  secondary: '#6192a1',
+  primary: '#6192A1',
+  secondary: '#A29060',
```
Comment (NoryDev) [inline src/ui/VavatoTheme.js:3]: The switching of these 2 will mean all blue/gold colors inverted in apps using the vavato-ui right?

**Review by NoryDev — APPROVED** [review]
Just one question, if answer is yes, maybe it's worth giving a heads-up to marketing automation.

Otherwise, 👏 very nice and clean code

File: src/ui/VavatoTheme.js (hunk 1 above)
Comment (cedricpim) [reply src/ui/VavatoTheme.js:3]: Yes, as mentioned here: https://www.figma.com/file/8Ov6NBiWMkByan9xg275bB/Vavato-Internal-team-library?node-id=0%3A1 (page 2)

**Comment by cedricpim** [comment]
> Just one question, if answer is yes, maybe it's worth giving a heads-up to marketing automation.
> 
> Otherwise, clap very nice and clean code

It will change only once they update vavato-ui library. And they don't do it often, they are using version 1.2 (we are currently at version 1.4.24).

But I can give them a heads-up. Where should I do it?

**Pull request merged**